Update math/index.rst and add math/img

gabrielgriesser opened the pull request.

Commit:
Ajout de weakref

JonathanGuerne committed:
commit test

JonathanGuerne committed:
Ajout de invoke

jojo4332 committed:
commit mise en place invoke remplace commit précédent

jojo4332 committed:
ajout du dossier invoke

jojo4332 committed:
simplification du fichier index pour compliation

Commit:
choix du sujet

Commit:
Sujet choisi

simojma3a committed:
Ajout de datetime.

johndacost committed:
test push

xiorcale committed:
Add introduction, logo and basic possibility of usage.

Signed-off-by: QVaucher <quentin.vaucher@protonmail.com>

xiorcale committed:
Add basic example.

Signed-off-by: QVaucher <quentin.vaucher@protonmail.com>

xiorcale committed:
Add drawing chapter.

xiorcale committed:
add second example.

xiorcale committed:
Add pyQt chapter

JonathanGuerne committed:
delete example for the JSON example

crypt0log-hai committed:
Create pycrypto

Commit:
pytest

Commit:
pytest

JonathanGuerne committed:
installation of invoke

charlyDevor committed:
Initialisation du sujet API REST HUG

JonathanGuerne committed:
installation and introduction of the invoke documentation

Commit:
Update math/index.rst and add math/img

73VW committed:
Résolution conflits

73VW committed:
Résolution conflits

greut commented:
:warning: en python on utilise des espaces (4) plutôt que des tabulations.

greut committed:
Sphinx en mode super-pénible.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

greut committed:
Correction des liens cassés.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

greut committed:
Ajout de pycrypto à l'introduction.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

johndacost committed:
Ajout du fichier de base UseItertools

johndacost committed:
Ajout du fichier de base UseItertools

charlyDevor committed:
Titre du document

kaser1996 committed:
Time

xiorcale committed:
add pillow doc to config file

xiorcale committed:
try to add ref to the doc

greut committed:
Utilisation de pycon plutôt que python.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

73VW committed:
Premier jet

johndacost committed:
chain, compress, count, map

johndacost committed:
chain, compress, count, map

johndacost committed:
itertools

johndacost committed:
refactor name

johndacost committed:
refactor name

johndacost committed:
ajout de la config + fonction compress

johndacost committed:
ajout de la config + fonction compress

johndacost committed:
Ajout de la fonction map et compress

johndacost committed:
Ajout de la fonction map et compress

johndacost committed:
ajout de nouvelles fonctions

- takewhile
- dropwhile
- filter, filterfalse

johndacost committed:
ajout de nouvelles fonctions

- takewhile
- dropwhile
- filter, filterfalse

greut committed:
pycodestyle, pydocstyle et isort, partout.

build.sh permet de lancer la construction comme sur Travis.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

greut committed:
Merci M. Friedli... je l'avais raté.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

xiorcale committed:
add link to doc.

73VW committed:
Continuation du projet

greut committed:
Nettoyage.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

johndacost committed:
clean project

73VW committed:
Corrections fautes d'orthographe

johndacost committed:
correction pycodestyle

greut committed:
Correction des exemples JSON.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

johndacost committed:
syntaxe pycon pour les exemples chain()

johndacost committed:
Revert "syntaxe pycon pour les exemples chain()"

This reverts commit 77c6c207fc9e23b8433e85fa47d9babc7582ca11.

johndacost committed:
Revert "Revert "syntaxe pycon pour les exemples chain()""

This reverts commit 13658dfaa161746f7b41c728d7e25828a8a8b365.

johndacost committed:
Revert "Revert "syntaxe pycon pour les exemples chain()""

This reverts commit 13658dfaa161746f7b41c728d7e25828a8a8b365.

xiorcale committed:
Add basic concepts and update links to the Pillow doc

greut committed:
Ajout de matplotlib.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

greut committed:
Nettoyage de README.rst

greut committed:
Intersphinx et exemples avec invoke.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

greut committed:
Exécution des scripts Python lors du build.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

greut committed:
Jonathan avec le h au bon endroit.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

greut commented on source/math/index.rst:
```diff
@@ -1,5 +1,124 @@
+
+.. code-block:: python3
+
+	from math import *
```
`from x import *` n'est pas recomandé.

greut committed:
Correction exemple JSON et test.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

xiorcale committed:
Add a conclusion and a chapter on bands.

xiorcale committed:
Spell correction.

xiorcale committed:
fix typo.

xiorcale committed:
fix typo.

xiorcale committed:
Add image from first example.

johndacost committed:
correction des exemples selon pycon

greut committed:
Corrections et todo's

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

johndacost committed:
fonctionnement de base des itérateurs

73VW committed:
Continuation du projet

Correction indenation, tabulation, ajout de l'origine de l'image dans sa
propriété target

73VW committed:
Résolution soucis d'indentation apparu lors de la conversion (oublié de vérifier avant de commit...)

greut committed:
Fichier externe pour l'exemple secrets.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

xiorcale committed:
Corrections

xiorcale committed:
Add PyQt example

xiorcale committed:
Improve PyQt example.

xiorcale committed:
Improve PyQt example.

xiorcale committed:
Remove magic numbers in every example. Improve PyQt example.

johndacost committed:
ajout de l'exemple

Commit:
first version of unittest without Mock and Doctest

Commit:
Corrected some bugs of test-fonction.py

Commit:
Docstring added

JonathanGuerne committed:
avancement dans l'ecriture du rapport

Commit:
deleted script python

Commit:
add link to unittest

greut committed:
Sortie de l'exemple unittest.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

greut committed:
Distribution d'étoiles.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

xiorcale committed:
Move color outside the loop in example2.

xiorcale committed:
improve example2.py and add intersphinx link in index.rst.

Commit:
new branch

xiorcale committed:
correct librairie to bibliothèque.

johndacost committed:
ajout de la conclusion + exemple générateur

greut committed:
Petit nettoyage.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

greut committed:
Pas mal de corrections...

ping @johndacost

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

xiorcale committed:
Improve basique example -> add description of L mode.

Commit:
extraction du code et remplacement des images par un lien

Commit:
first python N corrections

Commit:
seconde python N corrections

Commit:
third python N corrections

Commit:
4th python N corrections

Commit:
5th python N corrections

Commit:
6th python N corrections

Commit:
7th python N corrections

Commit:
8th python N corrections encodage

Commit:
9th python N corrections

greut committed:
PIL: Mise en forme de code

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

greut committed:
Ajout d'une étoile à PIL.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

greut committed:
msgpack et compression d'un gros fichier json.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

JonathanGuerne committed:
finalisation de la doc et ajout d'exemple pratique (génération de la doc html et checks)

JonathanGuerne committed:
corection probleme de syntaxe dans tasks.py

73VW committed:
Avancement du projet, ajout d'un exemple réaliste

Commit:
New section realised -> Doctest & some improvement of unittest

JonathanGuerne committed:
checks de la docs

JonathanGuerne committed:
correction de l'ordre des imports

73VW committed:
Modifications mineurs

JonathanGuerne committed:
correction d'erreur dans index.rst

73VW committed:
autre modifications mineures suiteau review

73VW committed:
modification du chapitre utilisation d'un token, ajout de précisions

Commit:
corrections des conseils de l'enseignant

Commit:
corrections des conseils de l'enseignant suite

Commit:
10th python N corrections + remove code.py

Commit:
11th python N corrections

greut committed:
Update index.rst

Sortie d'introduction.

Commit:
Update math/index.rst

greut commented on source/math/index.rst:
```diff
@@ -23,7 +22,7 @@ par exemple :``1/5 + 1/2 = 7/10``  sera affiché correctement.
 par exemple : ``1/2 + 1/3 = 5/6`` sera affiché sour la forme ``0.83333333333333``
 
-Pour effectuer des calculs exacts avec des fractions, on importe le module fractions.py 
+Pour effectuer des calculs exacts avec des fractions, on importe le module *FRACTIONS*
```
🙀 !

```rst
:py:mod:`fractions`
```

gabrielgriesser replied:
C'est à dire ...?

greut replied:
Pourquoi crier de la sorte : _FRACTIONS_ ?

greut commented on source/math/index.rst:
```diff
@@ -109,16 +107,165 @@ En Python, la priorité des opérations s'effectue comme en algère. On effectue
+	from fractions import *
+	a=Fraction(24,10)
+	print(a.numerator)
+	print(a.denominator)
```
sortez le code dans des fichiers externes. Pour un exemple aussi simple, utilisez `pycon`

```python
>>> from fractions import Fraction
>>> Fraction(24, 10)
12 / 5
```

greut commented on source/math/index.rst:
```diff
@@ -109,16 +107,165 @@ En Python, la priorité des opérations s'effectue comme en algère. On effectue
+	
+	from fractions import *
+	a=Fraction(24,10)
+	print(a+0.0)
```
Ceci est dég... vraiment pas beau!

KillianCastella committed:
premières modifs test

greut committed:
Ajout de todos, çà et là.

ping @andasilva

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

greut committed:
Add rstcheck checks.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

greut committed:
Un rstcheck un peu meilleur avec deux problèmes corrigés.

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

JonathanGuerne committed:
possiblement le dernier commit modification de la recuperation des exemples

Niklaus5665 committed:
Ajout image

JonathanGuerne committed:
correction d'erreur lors de l'affichage d'exemple dans la documentation

Niklaus5665 committed:
essai passage test

Niklaus5665 committed:
ajout tutorial

Niklaus5665 committed:
Ajout tutorial

Staminah committed:
Test

Niklaus5665 committed:
a

JonathanGuerne committed:
commit contenu final + correction orthographe

Staminah committed:
image

JonathanGuerne committed:
ajout de lien et propreté des exemples

JonathanGuerne committed:
suppression du fichier tasks.py dans le dossier exemple pour se baser sur le fichier tasks.py à la racine

greut committed:
invoke: petits nettoyages

ping @JonathanGuerne

Signed-off-by: Yoan Blanc <yoan@dosimple.ch>

Commit:
Update math/index.rst and add examples

Commit:
update math/index.rst and add example

greut commented:
Vous avez un problème quelque part au vu de la taille du diff : **+26,255 −4**

gabrielgriesser commented:
Autant pour moi, je n'avais pas fait de rebase

greut commented:
@gaboss222 côté rebase vous êtes bon.

greut commented:
![big diff](https://cloud.githubusercontent.com/assets/1388/24241387/f481b5e0-0fb4-11e7-8d68-938bea3bc2fa.png)

Commit:
update math/index.rst and add examples

Commit:
update math/index.rst and add examples

Commit:
resolve conflicts

Commit:
resolve conflicts, replace tab by space

Commit:
resolve conflicts for examples

Commit:
resolve conflicts for examples

Commit:
resolve conflicts for examples

Commit:
UNEXPECTD INDENTATION ?!

Commit:
Try to resolve UNEXPECTED INDENTATION !

gabrielgriesser commented:
Je veux bien faire un rebase, mais régler les 200aines de conflicts... il n'y a pas d'autres moyen ..?
![conflict](https://cloud.githubusercontent.com/assets/26295873/24245620/efbf3f00-0fc3-11e7-9ec9-7c9bda1aacb4.JPG)

gabrielgriesser commented:
à noter que l'image est un avant-goût de tout les conflicts

greut commented:
@gaboss222 00ecf6a est un merge et il faut la merde.

Solution simple, prenez vos fichiers et recréez une branche :
```console
$ cp -r source/math math
$ git checkout master
$ git pull origin  # origin étant HE-Arc
$ git checkout -b math
$ cp -r math source/
$ git add source/math
$ git commit
$ git push projet math
```

greut commented:
#117
